src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1495,12 +1491,12 @@ private static CanDoPathLookupResult CanDoPathLookup(string possiblePath)
         /// The enumerator that uses the Path to
         /// search for commands.
         /// </summary>
-        private CommandPathSearch _pathSearcher;
+        private CommandPathSearch? _pathSearcher;
```

Comment:
Perhaps there is an example in .Net Runtime repo how make this non-nullable.

Reply:
That is a feature under development, but not available in the current compiler.